grammar.js

```diff
@@ -2092,26 +2101,29 @@ module.exports = grammar({
       ),
       optional($.where),
       optional($.group_by),
+      optional($.window_clause),
```

Comment:
is is correct that the window_clause can be between group_by and order_by? Does the order matter here? I thought that is should be last.

Reply:
It does make sense as `limit` and `order by` apply to the result of set operations (union, intersect, except) & you can see those are a dividing line